test/integration/crud/bulk.test.js

```diff
@@ -22,6 +27,34 @@ describe('Bulk', function () {
   before(function () {
     return setupDatabase(this.configuration);
   });
+  describe('class BulkOperationBase', () => {
```

Comment:
small comment - based on the doc I sent I think we should use just the classname and then #raw for the method name